dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/TrackedEntityInstanceController.java

```diff
@@ -216,7 +218,7 @@ public void getAttributeImage(
                 unauthorized( "You're not authorized to access the TrackedEntityInstance with id: " + teiId ) );
         }
 
-        if ( attribute.size() == 0 )
+        if ( attribute.isEmpty() )
```

Comment:
I know it was this way before, but would it make more sense to rename it to `attributes`

Reply:
I think this was `attribute` because the list will always have either 0 or 1 object.

Reply:
otherwise it make sense to have `attributes` rather `attribute`